packages/api/tests/lib/rules/test-rulesHelpers.js

```diff
@@ -100,6 +101,19 @@ const getKinesisEventMappings = async () => {
   return await Promise.all(mappingPromises);
 };
 
+const waitForEventSourceMappingsDeletion = () =>
+  pWaitFor(
+    async () => {
+      const eventMappings = await getKinesisEventMappings();
+      return (eventMappings.map((eventMapping) => eventMapping.EventSourceMappings)
+        .flat().length === 0);
+    },
+    {
+      interval: 3000,
+      timeout: 60 * 1000,
+    }
+  );
+
```

Comment:
localstack is updated to do what aws-sdk doc says `When you delete an event source mapping, it enters a Deleting state and might not be completely deleted for several seconds.`